demisto_sdk/commands/common/tools.py

```diff
@@ -1275,3 +1275,26 @@ def camel_to_snake(camel: str) -> str:
     camel_to_snake_pattern = re.compile(r'(?<!^)(?=[A-Z][a-z])')
     snake = camel_to_snake_pattern.sub('_', camel).lower()
     return snake
+
+
+def get_support_level_from_file_path(file_path: str) -> str:
```

Comment:
seems like a good tool which is similar to what is used in `base_validator.py` in the `update_checked_flags_by_support_level` method - we might want to think about merging the two ideas